README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
+
 To run all the tests:
 
     $ bundle exec rspec
 
 See RSpec [docs](https://relishapp.com/rspec/rspec-core/docs/command-line) for
-more information.
+more information. 
```

Comment:
Or at this to your .vimrc:

```
" Remove tailing whitespace
nnoremap <leader>w :%s/\s\+$//<cr>:let @/=''<cr>
```

This makes <leader>-w (normally \w) remove all trailing whitespace for the current buffer.